baldrick/github/github_api.py

```diff
@@ -777,9 +797,10 @@ def get_repo_config(self, branch=None, path_to_file='pyproject.toml'):
             Configuration parameters.
 
         """
+        upstream = RepoHandler(self.base_repo_name)
         if not branch:
-            branch = self.base_branch
-        return super().get_repo_config(branch=branch, path_to_file=path_to_file)
+            branch = upstream.default_branch
```

Comment:
Aside from my comments in #111 this change will probably fix the bug (although I don't know why it's needed). We should however, still be using base branch; we always want to pull the config from the branch the PR is targeting, to allow varying configs over different branches.

```suggestion
            branch = upstream.base_branch
```